tests/test_extractor.py

```diff
@@ -75,17 +81,22 @@ def teardown(self):
             bytes_io.close()
 
     def ipynb_file(self):
-        return open(f'{SAMPLES_DIR}/{self.IPYNB_NAME}', encoding='utf-8')
+        return open(Path(SAMPLES_DIR) / self.IPYNB_NAME, encoding='utf-8')
 
     @staticmethod
     def py_files(filenames: Iterator[str]):
         for file_name in filenames:
-            yield open(f'{SAMPLES_DIR}/{file_name}')
+            yield open(Path(SAMPLES_DIR) / file_name)
+
+    @staticmethod
+    def get_bytes_io_file(file_name) -> BytesIO:
+        with open(Path(SAMPLES_DIR) / file_name, 'br') as open_file:
+            return BytesIO(open_file.read()), file_name
```

Comment:
A neat trick: You can use `(Path(SAMPLES_DIR) / file_name).read_bytes()`

Reply:
Tried it and it doesn't work. I really need the `BytesIO` object